test/integration_test/basic_test.go

```diff
@@ -239,6 +241,14 @@ var testStorageClusterBasicCases = []types.TestCase{
 			return ci_utils.PxOperatorVersion.LessThan(ci_utils.PxOperatorVer1_8)
 		},
 	},
+	{
+		TestName:        "InatallWithCustomPrometheusSpecParams",
```

Comment:
can we change it to `BasicPrometheusRegression`? so in the future we can add more steps inside.